chia/full_node/mempool_manager.py

```diff
@@ -714,6 +742,9 @@ async def new_peak(
             self.mempool = Mempool(old_pool.mempool_info, old_pool.fee_estimator)
             self.seen_bundle_hashes = {}
 
+            # Reset the singleton fast forward puzzle hash to unspent lineage info map
+            self.puzzle_hash_to_unspent_lineage_info = {}
```

Comment:
this will re-trigger, possibly a bunch of, database lookups, for the lineage info, right?